pkg/controller/gitjob/gitjobs.go

```diff
@@ -173,6 +174,11 @@ func (h Handler) generateJob(obj *v1.GitJob) (*batchv1.Job, error) {
 		Spec: obj.Spec.JobSpec,
 	}
 
+	ttl := int32(defaultTTL)
```

Comment:
can we move the `ttl` var inside of the if as it is not used outside?

Reply:
Oh yes